src/components/Shared/Tables/SimpleTableComponent.tsx

```diff
@@ -15,6 +15,8 @@ export class SimpleTableComponentProps {
     isIndexZero?: boolean;
     boldIndex?: number[];
     updateTableRef?: (ref: Table) => void;
+    isTooltip?: boolean;
```

Comment:
Just a minor suggestion, but perhaps we can use just `tooltipIndex` and check `typeof this.props.tooltipIndex !== "undefined"` so that we don't need to set an additional `isTooltip` boolean.

Reply:
Agree.
The suggestion is accomplished with the latest commit 283d4cd.